mini-mongoose.js

```diff
@@ -0,0 +1,44 @@
+ [
```

Comment:
WHAT does that even mean.

On Fri, Oct 16, 2015 at 10:54 AM, tgaff <notifications@github.com> wrote:

> In mini-mongoose.js
> <https://github.com/sf-wdi-22-23/23-homework/pull/103#discussion_r42270598>
> :
>
> > @@ -0,0 +1,44 @@
> > + [
>
> Oh no! [image: :whale:] @isomdurm <https://github.com/isomdurm> all your
> files need to go into your directory!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sf-wdi-22-23/23-homework/pull/103/files#r42270598>.
>